tests/sentry/api/serializers/test_apitoken.py

```diff
@@ -0,0 +1,49 @@
+from sentry.api.serializers import ApiTokenSerializer
+from sentry.testutils.cases import TestCase
+
+
+class TestApiTokenSerializer(TestCase):
+    def setUp(self) -> None:
+        self._user = self.create_user()
```

Comment:
`self.user` is automatically set in `TestCase` so you can just use that directly in your tests which is convenient.

Same thing for `self.project`, `self.team`, and `self.organization`. The only difference is I think those are lazy so they only get created after you call it in the test.

Reply:
This is good to know, thank you!